clang/test/ClangScanDeps/Inputs/modules_cdb_by_mod_name.json

```diff
@@ -0,0 +1,22 @@
+[
+{
+  "directory": "DIR",
+  "command": "clang -E -IInputs -D INCLUDE_HEADER2 -MD -MF DIR/modules_cdb2.d -fmodules -fcxx-modules -fmodules-cache-path=DIR/module-cache -fimplicit-modules -fimplicit-module-maps -gmodules -x c++",
+  "file": ""
+},
+{
+  "directory": "DIR",
+  "command": "clang -E -IInputs -fmodules -fcxx-modules -fmodules-cache-path=DIR/module-cache -fimplicit-modules -fimplicit-module-maps -x c++",
+  "file": ""
+},
+{
+  "directory": "DIR",
+  "command": "clang -E -IInputs -fmodules -fcxx-modules -fmodules-cache-path=DIR/module-cache -fimplicit-modules -fimplicit-module-maps -o a.o -x c++",
+  "file": ""
+},
+{
+  "directory": "DIR",
+  "command": "clang -E -IInputs -fmodules -fcxx-modules -fmodules-cache-path=DIR/module-cache -fimplicit-modules -fimplicit-module-maps -o b.o -x c++",
```

Comment:
Why are there so many compile commands in this file when the test only scans dependencies of a single module?